Import a Status Community (#537)
status-im/help.status.im: docs/en/status-communities/import-a-status-community.md

---
id: 445
revision: 0
language: en
hide:
  - navigation
---

> # Import a Status Community

Reviewer: The "Import community" button is not easy for mobile users to see, users may not know they can import a community. Hence, they may not want to read "import a community" or search for it. 
How about changing the title to something like "View a community that's shared with you"

Author: Thanks, @cheny0 - I'm rewriting and restructuring the content based on your feedback. 

It wasn't easy to find an article's name. The one you proposed makes sense, but it's a passive sentence, which I prefer to avoid for titles. At the same time, I want to avoid using the "Join" word because the user is not joining the community yet; it's only adding/importing it. The "join" part comes later, as long as s/he is accepted and meets the community requirements.

I ended up with "Import a community someone shares with you".

> To import a Status Community, you use the community's public key. This key is a large string of characters that helps you find the community you want to join. To make sharing and importing communities easier, Status includes the community's public key in a QR code or link, like this one: `https://join.status.im/c/zQ3shMtRomXyM5svciMpdK7zDaR12XDeQWbzi4Fb8E6YywDrE`.

Reviewer: I'd add a sentence introducing in what cases users will want to import a community, for example, "If someone shares a Status Community with you, you can import the community to quickly find it".
Because if without context, I'd assume this is what a community owner can do.

Author: Thanks, @cheny0 -- I include this information in the paragraph right after that one ;)

> Importing a community can be helpful when you can't find the community or someone shares the community QR code or link.

=== "Mobile"

    1. From the tab, tap :mobile-communities: **Communities**.
    1. Follow these steps using the community QR code or link:
    
    | Import from | Steps |
    |:---|:---|
    | QR code | 1. From the top navigation, tap :mobile-scan: **Scan** and scan the QR code.</br>2. When the community page opens, tap :mobile-open-lock: **Join community**. |
    | Link | 1. Tap :mobile-add: **Add** > :mobile-receive: **Import community**.</br>2. In the **Community key** text field, enter the community link or public's community key.</br>3. Verify the community you're about to import and tap :mobile-receive: **Import community**.</br>4.  When the community page opens, tap :mobile-open-lock: **Join community**. |

=== "Desktop"

    1. From the navigation sidebar, click :desktop-communities: **Communities**.
    1. On top of the content area, click **Import using key**.
    1. In the **Community key** text field, enter the community link or public's community key.
    1. Click **Import**. If the **Import** option is unavailable, verify you're entering the correct key.

!!! tip

>     After opening the link or scanning the QR code, people can [install Status][download-status] if they haven't already.

Reviewer: ```suggestion
    After opening the link or scanning the QR code, you can [install Status][download-status] if you haven't already.
```
